PROCESSORS.md

```diff
@@ -757,7 +757,8 @@ Merges a Group of FlowFiles together based on a user-defined strategy and packag
 In the list below, the names of required properties appear in bold. Any other properties (not in bold) are considered optional. The table also indicates any default values, and whether a property supports the NiFi Expression Language.
 
 | Name | Default Value | Allowable Values | Description | 
-| - | - | - | - | 
+| - | - | - | - |
+|Attribute Strategy|Keep Only Common Attributes|Keep Only Common Attributes<Br>Keep All Unique Attributes<Br>|Determines which FlowFile attributes should be added to the bundle. If 'Keep All Unique Attributes' is selected, any attribute on any FlowFile that gets bundled will be kept unless its value conflicts with the value from another FlowFile. If 'Keep Only Common Attributes' is selected, only the attributes that exist on all FlowFiles in the bundle, with the same value, will be preserved.| 
```

Comment:
What happens to an attribute which exists in two flow files with different values if 'Keep All Unique Attributes' is selected?  Will both be discarded, or will one of them be kept?

Reply:
Both will be discarded

Reply:
I would add that clarification to the description: "... unless its value conflicts with the value from another FlowFile (in which case neither, or none, of the conflicting attributes will be kept)."

Reply:
Shall I also update the property in MergeContent.cpp as currently it has the same description?

Reply:
I would, unless there is a length limit in the C2 GUI.